### PR TITLE
Web/performance

### PR DESCRIPTION
**Pull request by conorato**
Update performance page content

### CODE REVIEW

File: web/src/views/performance/index.js [hunk 1]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
         description={text['header_description']}
       />
       <Container className="mt-5 text-justify">
-        <p>
-          Ever wondered what is the value of this application? This page aims to illustrate the relative performance of
-          our sleep scoring compared to clinical hypnogram scoring (which is usually considered the state-of-the-art
-          technique).
+        <p className="lead">
+          This page aims to illustrate the performance of our sleep scoring algorithmcompared to manual and professional
```
Comment (WilliamHarvey97) [inline web/src/views/performance/index.js]:
```suggestion
          This page aims to illustrate the performance of our sleep scoring algorithm compared to manual and professional
```

File: web/src/views/performance/index.js [hunk 2]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
+          scoring, which is considered the gold standard in sleep stage scoring. If you want to learn more about how
```
Comment (WilliamHarvey97) [inline web/src/views/performance/index.js:61]:
```suggestion
          manual scoring, which is considered the gold standard in sleep stage scoring. If you want to learn more about how
```

File: web/src/views/performance/index.js [hunk 3]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
-        <p>
-          Ever wondered what is the value of this application? This page aims to illustrate the relative performance of
-          our sleep scoring compared to clinical hypnogram scoring (which is usually considered the state-of-the-art
-          technique).
+        <p className="lead">
+          This page aims to illustrate the performance of our sleep scoring algorithm compared to professional manual
+          scoring, which is considered the gold standard in sleep stage scoring. If you want to learn more about how
+          we've defined our sleep scoring algorithm, please either refer to our presentation video in our home page, or
+          to our{' '}
+          <a href="https://github.com/PolyCortex/polydodo/wiki/Model" target="_blank" rel="noreferrer">
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:64]:
```suggestion
          <a href="https://github.com/PolyCortex/polydodo/wiki/Classification-Model-&-Pipeline" target="_blank" rel="noreferrer">
```

File: web/src/views/performance/index.js [hunk 4]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
-            This subset is composed of full nights of sleep coming from five subject of a different age group.&nbsp;
-          </li>
-          <li>
-            Then, we will check how this classifier performs on a full night recorded on one of our members. In order to
-            be able to make comparisons, we ask for the help of a medical electrophysiologist to score our data. This
-            manual scoring will serve as reference to get an idea of the accuracy of our model on data acquired using an
-            OpenBCI under non-clinical conditions. The AASM manual was used for scoring.
+            Then, we will check how this classifier performs on a full night recorded on one of our members. It is the
+            night displayed in preview mode. In order to be able to make comparisons, we asked for the help of a medical
+            electrophysiologist to score our data. This manual scoring will serve as reference to get an idea of the
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:78]:
```suggestion
            electrophysiologist to score our data. This manual scoring will serve as reference to get an idea of our
```

File: web/src/views/performance/index.js [hunk 5]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
+            accuracy of our model on data acquired using an OpenBCI under non-clinical conditions.
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:79]:
```suggestion
            model's accuracy on data acquired using an OpenBCI under non-clinical conditions.
```

File: web/src/views/performance/index.js [hunk 6]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
           </li>
         </ul>
-        <h3 className="mt-5">Classifier's accuracy according to Sleep-EDF</h3>
+        <p className="lead">
+          We will finally cover the limitations of our current sleep staging model and further work to be done in order
+          to improve our results.
+        </p>
+        <h3 className="my-5">Model training and selection</h3>
+        <p>
+          In order to train our model to predict sleep stages based on EEG data, we used an open source dataset,
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:93]:
```suggestion
          In order to train our model for the prediction of sleep stages based on EEG data, we used an open source dataset,
```

File: web/src/views/performance/index.js [hunk 7]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
+        <p>
+          The test set, on which these metrics were calculated, is composed of randomly chosen subjects from different
+          ages groups (a 33 year old female, a 54 year old female, a 67 year old female and a 88 year old male), so that
+          the obtained score is the most representative of our ability to classify sleep, no matter the age.
+          {/* On another side, we could compare the results obtained to the ones found in the litterature. To do so, we had to find a
+          paper that uses the same dataset, the same metric and that splits their dataset in a similar fashion as
+          ours.TODO. */}
+        </p>
+        <p>
+          Although we obtain good results, it didn't quite validate that our classifier could accurately score OpenBCI
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:146]:
```suggestion
          Although we obtained good results, it didn't quite validated that our classifier could accurately score OpenBCI
```

Pas sûr ??

File: web/src/views/performance/index.js [hunk 8]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
+          data into sleep stages. After all, we only validated on data coming from the same acquisition hardware than
+          the data we've trained on, which is not the case when we analyze data submitted in the application. We then
+          had to make our own polysomnographic dataset based on the hardware we use, namely an OpenBCI board.
+        </p>
+        <h3 className="my-5">Manual scoring of OpenBCI data and comparison to our classifier</h3>
+        <p>
+          As we had limited resources, we had in mind to create a small dataset of two manually scored night's of sleep,
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:153]:
```suggestion
          As we had limited resources, we planned to create a small dataset of two manually scored night's of sleep,
```

File: web/src/views/performance/index.js [hunk 9]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
+        <h3 className="mt-5">Classifier's accuracy according to Sleep-EDF</h3>
         <D3Component
-          callback={(svg, data) => createComparativeHypnogram(svg, data, ['Electrophysiologist', 'Sleep-EDF'])}
-          data={[electrophysiologistWoman78yoSleepEDF.epochs, physionetWoman78yoSleepEDF.epochs]}
+          callback={(svg, data) => createComparativeHypnogram(svg, data, ['Classifier', 'Sleep-EDF'])}
+          data={[predictedWoman78yoSleepEDF.epochs, physionetWoman78yoSleepEDF.epochs]}
         />
+        <p className="my-5">
+          We can see that some differences between the automatic classification and SleepEDF's are the same as
+          Alexandra's. Per example, near the end of the night, both Alexandra and the automatic scoring model classified
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:256]:
```suggestion
          Alexandra's. For instance, near the end of the night, both Alexandra and the automatic scoring model classified
```

File: web/src/views/performance/index.js [hunk 10]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
+            ['N3', 100, 34, 51, 91],
+            ['Accuracy', '', '', 76, 1188],
           ]}
         />
+        <h3 className="my-5">Limitations and further development</h3>
+        <p>
+          If we take a step back and look at the main differences between the automatic and manual scoring that
+          Alexandra did, there's notably the manual used for the classification. Indeed, the dataset we've used to train
+          our model contains sleep stages classification based on the 1968 Rechtschaffen and Kales manual, whereas
+          Alexandra, of course, used the most recent manual scoring guide, whereas the American Academy of Sleep
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:276]:
```suggestion
          Alexandra, of course, used the most recent manual scoring guide, which is the American Academy of Sleep
```

File: web/src/views/performance/index.js [hunk 11]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
+          Medicine Manual for the Scoring of Sleep and Assoicated Events. In order to output AASM's sleep stages instead
+          of R&K's sleep stages, we've merged both Sleep Stage 3 and 4 together. Further work could be done to either
+          look at the litterature to see if there's a better way to translate R&K's sleep stages into AASM's sleep
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:279]:
```suggestion
          review the litterature to see if there's a better way to translate R&K's sleep stages into AASM's sleep
```

File: web/src/views/performance/index.js [hunk 12]
```diff
@@ -56,77 +56,255 @@ const Performance = () => {
...
+          stages. Even better, it would be to train on labels scored based on the latest AASM's sleep stages. We have
+          considered more recent datasets, such as{' '}
+          <a href="http://www.ceams-carsm.ca/mass" target="_blank" rel="noreferrer">
+            the Montreal Archive of Sleep Studies (MASS)
+          </a>
+          , but it involved having a complete accreditation coming from an ethic board.
```
Comment (cicero-mt) [inline web/src/views/performance/index.js:285]:
```suggestion
          , but it involved having a complete accreditation coming from an ethics board.
```

**Pull request merged**